.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
```

Comment:
1. why are the playwright tests their own separate workflow rather than part of an existing one?
2. why do we only want to run them on main and master? also why are we referencing a master branch since we don't use one

Reply:
These are not final version of the playwright files. they are just a skeleton to setup for now and waiting for @harrim91 to finalise the main files

Reply:
Lets get rid of the reference to master as we'll never have both branches.  Happy for this to be a placeholder that will be picked up by @sean-corrigan1 when we have worked out our processes for testing.